### PR TITLE
Add img and mediaUpload fields to collection

### PR DESCRIPTION
**Pull request by subarashii-huch09**
**Changes made in this PR:**

- Added `mediaUploadId` field to collection
- Updated the example request and example response

### CODE REVIEW

File: legacy/src/api/assets/tokens.md [hunk 1]
```diff
@@ -28,18 +28,20 @@ A redemption condition is created for each merchant that accepts tokens from a c
...
+| maxValue          | {% dt Monetary %} {% opt %}               | The maximum agreed value that any merchants will be settled for a token redemption.         |
+| id                | String                                    | The token collection id                                                                     |
+| test              | Boolean                                   | `true` if the token collection is for testing purposes only.                                |
+| status            | String                                    | The status of the token collection. Valid values include 'active'.                          |
+| createdBy         | {% dt CRN %}                              | The identity that created the activity.                                                     |
+| createdAt         | {% dt Timestamp %}                        | Timestamp at which the token collection was created.                                        |
+| mediaUploadId     | String {% opt %}                          | The id of the media upload image used for all tokens that are created from this collection. |
```
Comment (Ray2126) [inline legacy/src/api/assets/tokens.md:43]: Can you link to the Media Upload page here?

Comment (Ray2126) [inline legacy/src/api/assets/tokens.md:43]: It's actually the `img` that will be used for all tokens created from the collection.

Comment (Ray2126) [inline legacy/src/api/assets/tokens.md:43]:
```suggestion
| mediaUploadId     | String {% opt %}                          | The id of the media upload image of the collection. |
```

**Review by Ray2126 — CHANGES_REQUESTED** [review]

File: legacy/src/api/assets/tokens.md (hunk 1 above)
Comment (subarashii-huch09) [reply legacy/src/api/assets/tokens.md]: Yep, I've added the link :)

File: legacy/src/api/assets/tokens.md [hunk 2]
```diff
@@ -110,6 +113,8 @@ A redemption condition is created for each merchant that accepts tokens from a c
 	"status": "active",
 	"createdBy": "crn::user:b657195e-dc2f-11ea-8566-e7710d592c99",
 	"createdAt": "2021-05-12T04:30:11.001Z",
+	"mediaUploadId": "12345",
+	"img": "https://static.centrapay.com/assets/collections/{mediaUploadId}.png"
```
Comment (Ray2126) [inline legacy/src/api/assets/tokens.md:117]: Let's change this `img` URL to a more realistic one :)

**Review by Ray2126 — APPROVED** [review]

File: legacy/src/api/assets/tokens.md (hunk 2 above)
Comment (subarashii-huch09) [reply legacy/src/api/assets/tokens.md:117]: @Ray2126  something like this ?
https://static-assets-bucket-centrapay.s3.ap-southeast-1.amazonaws.com/{name}.png

**Comment by subarashii-huch09** [comment]
@Ray2126  I've removed `img` field, only keeping `mediaUploadId`

**Review by Ray2126 — APPROVED** [review]

**Pull request merged**